src/regex.js

```diff
@@ -45,6 +45,8 @@ SVG.regex = {
   // split at whitespace and comma
 , delimiter:        /[\s,]+/
 
+, viewbox:          /[+-]?(?:\d+(?:\.\d*)?|\.\d+)(?:e[+-]?\d+)?/gi
```

Comment:
I think we really can replace this one with the normal delimiter regex. This regex matches 4 numbers with optional exponent. But we can just do `split(/[,\s]+/)` instead which is used in all the other cases. So I will remove this one